extensions/dolar-cripto-ar/src/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { useEffect, useState } from "react";
+import { MenuBarExtra } from "@raycast/api";
+import { LocalStorage } from "@raycast/api";
+import { useFetch } from "@raycast/utils";
+
+type Dollar = { name: string };
+
+type DollarResponse = {
+  blue: { ask: number };
+  mep: { al30: { "24hs": { price: number } } };
+  ccl: { al30: { "24hs": { price: number } } };
+};
+
+const useRates = () => {
+  const [state, setState] = useState<{ dollar: Dollar[]; crypto: Dollar[]; isLoading: boolean }>({
+    dollar: [],
+    crypto: [],
+    isLoading: true,
+  });
+
+  useEffect(() => {
+    setState({
+      dollar: [{ name: "Blue" }, { name: "MEP" }, { name: "CCL" }],
+      crypto: [{ name: "BTC" }, { name: "ETH" }],
+      isLoading: false,
+    });
+  }, []);
+
+  return state;
+};
+
+export default function Command() {
+  const [selectedCurrency, setSelectedCurrency] = useState<string | undefined>();
```

Comment:
You might want to use [useCachedState](https://developers.raycast.com/utilities/react-hooks/usecachedstate) from `@raycast/utils` instead (and set the default value directly